playbooks/roles/ocp-upgrade/tasks/main.yaml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Upgrade OCP cluster
+  when: upgrade_image != ""
```

Comment:
Is it better to move the condition to playbook level and not use block here? Not sure what is a better approach.

Reply:
Done